src/MaterialDesignControls/ControlsMaterial3/BaseMaterialFieldControl.xaml.cs

```diff
@@ -0,0 +1,687 @@
+
+        public string LabelFontFamily
+        {
+            get { return (string)GetValue(LabelFontFamilyProperty); }
+            set { SetValue(LabelFontFamilyProperty, value); }
+        }
+
+        public static readonly BindableProperty LabelMarginProperty =
+            BindableProperty.Create(nameof(LabelMargin), typeof(Thickness), typeof(BaseMaterialFieldControl), defaultValue: new Thickness(16, 0, 16, 0));
+
+        public Thickness LabelMargin
+        {
+            get { return (Thickness)GetValue(LabelMarginProperty); }
+            set { SetValue(LabelMarginProperty, value); }
+        }
+
+        #endregion LabelText
+
+        #region SupportingText
+
+        public static readonly BindableProperty SupportingTextProperty =
+            BindableProperty.Create(nameof(SupportingText), typeof(string), typeof(BaseMaterialFieldControl), defaultValue: null, validateValue: OnSupportingTextValidate);
+
+        public string SupportingText
+        {
+            get { return (string)GetValue(SupportingTextProperty); }
+            set { SetValue(SupportingTextProperty, value); }
+        }
+
+        public static readonly BindableProperty SupportingTextColorProperty =
+            BindableProperty.Create(nameof(SupportingTextColor), typeof(Color), typeof(BaseMaterialFieldControl), defaultValue: Color.Gray);
+
+        public Color SupportingTextColor
+        {
+            get { return (Color)GetValue(SupportingTextColorProperty); }
+            set { SetValue(SupportingTextColorProperty, value); }
+        }
+
+        public static readonly BindableProperty SupportingSizeProperty =
+            BindableProperty.Create(nameof(SupportingSize), typeof(double), typeof(BaseMaterialFieldControl), defaultValue: Font.Default.FontSize);
+
+        public double SupportingSize
+        {
+            get { return (double)GetValue(SupportingSizeProperty); }
+            set { SetValue(SupportingSizeProperty, value); }
+        }
+
+        public static readonly BindableProperty SupportingFontFamilyProperty =
+            BindableProperty.Create(nameof(SupportingFontFamily), typeof(string), typeof(BaseMaterialFieldControl), defaultValue: null);
+
+        public string SupportingFontFamily
+        {
+            get { return (string)GetValue(SupportingFontFamilyProperty); }
+            set { SetValue(SupportingFontFamilyProperty, value); }
+        }
+
+        public static readonly BindableProperty SupportingMarginProperty =
+            BindableProperty.Create(nameof(SupportingMargin), typeof(Thickness), typeof(BaseMaterialFieldControl), defaultValue: new Thickness(16, 4, 16, 0));
+
+        public Thickness SupportingMargin
+        {
+            get { return (Thickness)GetValue(SupportingMarginProperty); }
+            set { SetValue(SupportingMarginProperty, value); }
+        }
+
+        #endregion SupportingText
+
+        #region CornerRadius
+
+        public static readonly BindableProperty CornerRadiusProperty =
```

Comment:
Apply same comment here, we can use only one bindable property with the same type than the Margin and Padding properties.